src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -49,6 +52,7 @@ export class StudentExamDetailComponent implements OnInit {
             this.course = courseResponse.body!;
         });
         this.student = this.studentExam.user;
+        this.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.course);
```

Comment:
I guess this has to be done inside the subscription as follows: 
``` 
this.courseService.find(this.courseId).subscribe((courseResponse) => {
    this.course = courseResponse.body!;
    this.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.course);
});
```
Otherwise line 55 gets executed before line 52 with the result that in the execution of line 55 `this.course` is still  undefined. This leads to `this.isAtLeastInstructor` becoming undefined.